src/components/screens/forms/forms-screen.tsx

```diff
@@ -9,105 +9,173 @@ import NotificationsActiveIcon from "@mui/icons-material/NotificationsActive";
 import NotificationsIcon from "@mui/icons-material/Notifications";
 import { AdminFormListStack, AdminFormTypographyField } from "styled/react-components/react-components";
 import { Link } from "react-router-dom";
+import { useApiClient } from "app/hooks";
+import Api from "api";
+import { Metaform, Reply } from "generated/client";
 
-const columns: GridColDef[] = [
-  {
-    field: "id",
-    headerName: strings.navigationHeader.formsScreens.formScreen.form.form,
-    flex: 1,
-    renderHeader: params => {
-      return (
-        <AdminFormListStack direction="row">
-          <ListIcon style={ { fill: "darkgrey" } }/>
-          <AdminFormTypographyField sx={{ fontWeight: "bold" }}>{ params.colDef.headerName }</AdminFormTypographyField>
-        </AdminFormListStack>
-      );
+interface Row {
+  id: string;
+  latestReply: string;
+  newReply?: string;
+}
+
+/**
+ * Forms screen component
+ */
+const FormsScreen: React.FC = () => {
+  const apiClient = useApiClient(Api.getApiClient);
+  const { metaformsApi, repliesApi } = apiClient;
+
+  const columns: GridColDef[] = [
+    {
+      field: "id",
+      headerName: strings.navigationHeader.formsScreens.formScreen.form.form,
```

Comment:
Maybe change string name to something like that so it would be more describing
```suggestion
      headerName: strings.navigationHeader.formsScreens.formScreen.formListing.form,
```